app/pkcs.py

```diff
@@ -20,23 +20,26 @@ class AlgorithmIdentifier(Sequence):
 
 
 class pkcs:
-    pkcs11 = None
+    DEFAULT_LIB_LOCATION = "/usr/lib64/libykcs11.so.2"
+    DEFAULT_HOMEBREW_LOCATION = "/opt/homebrew/lib/libykcs11.dylib"
+
```

Comment:
I do not think these are used anymore as this is moved to the `pkcs_lib_finder`.